JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R

```diff
@@ -286,7 +286,8 @@ PlotRobustnessSequential <- function(
       idx <- findInterval(val, c(1, 3, 10, 30, 100), rightmost.closed = FALSE)
       evidenceLevel <- c("Anecdotal", "Moderate", "Strong", "Very~Strong", "Extreme")[idx]
       
-      if (BF > 1 && bfType == "BF01" || BF < 1 && bfType != "BF01")
+      BF01 <- 1 / BF
```

Comment:
Would this break if someone supplied `BF = 1 / BF10, bfType = "BF01"`?
If so, it might be good to remove `bfType` and rename `BF` to `BF10` -- that way it becomes a little more likely that people will continue to supply only BF10's